tokio-stream/src/wrappers/broadcast.rs

```diff
@@ -0,0 +1,61 @@
+use crate::Stream;
+use async_stream::stream;
+use std::pin::Pin;
+use tokio::sync::broadcast::error::RecvError;
+use tokio::sync::broadcast::Receiver;
+
+use std::fmt;
+use std::task::{Context, Poll};
+
+/// A wrapper around [`Receiver`] that implements [`Stream`].
+///
+/// [`Receiver`]: struct@tokio::sync::broadcast::Receiver
+/// [`Stream`]: trait@crate::Stream
+pub struct BroadcastStream<T: Clone> {
+    inner: Pin<Box<dyn Stream<Item = Result<T, BroadcastStreamRecvError>> + Send + Sync>>,
+}
+
+/// An error returned from the inner stream of a [`BroadcastStream`].
+#[derive(Debug, PartialEq)]
+pub enum BroadcastStreamRecvError {
```

Comment:
It should be re-exported so the type is visible to the user.

Reply:
I added `pub use broadcast::BroadcastStreamRecvError;` in the `wrappers.rs`